cmd/flux/get_image_policy.go

```diff
@@ -74,7 +74,7 @@ func init() {
 func (s imagePolicyListAdapter) summariseItem(i int, includeNamespace bool, includeKind bool) []string {
 	item := s.Items[i]
 	status, msg := statusAndMessage(item.Status.Conditions)
-	return append(nameColumns(&item, includeNamespace, includeKind), item.Status.LatestImage, status, msg)
+	return append(nameColumns(&item, includeNamespace, includeKind), item.Status.LatestRef.Name, status, msg)
 }
 
 func (s imagePolicyListAdapter) headers(includeNamespace bool) []string {
```

Comment:
We also need to change the column names from:

```go
headers := []string{"Name", "Latest image", "Ready", "Message"}
```

To:

```go
headers := []string{"Name", "Image", "Tag", "Ready", "Message"}
```

This matches what we did in the kubectl printer columns here:

https://github.com/fluxcd/image-reflector-controller/blob/fe23c393a458eef24c4d7a100efd47a695e58e8c/api/v1beta2/imagepolicy_types.go#L204-L205